corehq/toggles.py

```diff
@@ -842,6 +842,13 @@ def _enable_search_index(domain, enabled):
     namespaces=[NAMESPACE_DOMAIN],
 )
 
+CASE_API_V0_6 = StaticToggle(
+    'case_api_v0_6',
+    'Enable the v0.6 Case API',
+    TAG_SOLUTIONS_LIMITED,
+    namespaces=[NAMESPACE_DOMAIN],
```

Comment:
Optional thought: if the intention of the flag is just to allow testing, it may make sense to also make it available to users in addition to domains, to prevent testers from having to enable it domain-wide. However, if the intention is that this will be in a limited release indefinitely, then having it be scoped to domains seems safer.

Reply:
That would make testing easier, though I do also intend for it to be in limited release for a while, so I'm inclined to leave it.